content/guides/concepts/pinning.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Pinning"
+menu:
+    guides:
+        parent: concepts
+---
+
+## Summary
+
+IPFS nodes treat the data they store like a cache, meaning that there is no guarantee that the data will continue to be stored.  Pinning a CID tells an IPFS server that the data is important and mustn't be thrown away.
+
+You should pin any content you consider important, to ensure that content is retained long-term. Since data important to someone else may not be important to you, pinning lets you have control over the disk space used by your IPFS nodes.
+
+## Context 
+
+An IPFS node can store data based on different kinds of user events.  For instance, a person may add a file with `ipfs add ...`.  Or a piece of unknown IPFS content can be requested, and the node can consult with its peers to find this data - the result will be stored in the local cache.  These data can use different policies for managing the cache - `ipfs add ...` will automatically pin the content.  Other IPFS commands do not include automatic pinning.
+
+When garbage-collection is triggered on a node, any pinned content is automatically exempt from deletion.  Non-pinned data is otherwise eligible to be deleted to save space (if requested, this data can be re-retrieved from another node. 
+
+## Pinning Services
+
+If you don't want to run your own IPFS node, you can use a pinning service.  Such a service normally trades money for the service of guaranteeing they'll keep your data pinned.
```

Comment:
> If you don't want to run your own IPFS node

I feel like this is a little off. You should still be running IPFS for lots of other reasons, even if you aren’t pinning much data locally. Some better example cases here might be:

- You don’t have a lot of disk space, but you want to ensure some data sticks around
- Your computer is a laptop or phone or a tablet that will have intermittent connectivity to the network, but you want to be able to access your file on IPFS from anywhere at any time, even when the device you added it from is offline.
- You want a backup that ensures your data is always available from another computer on the network in case you accidentally delete or garbage-collect on your own computer.